src/tools/pub/cmd/pub-get.md

```diff
@@ -143,6 +143,17 @@ command-line argument, as mentioned above.
 For options that apply to all pub commands, see
 [Global options](/tools/pub/cmd#global-options).
 
+### `--dry-run` or `-n`
+
+With this, pub reports the dependencies that would
```

Comment:
I'm not sure the full intention here. Reports the dependencies that would _what_?